test/tests/parallel/set1/spam_virus/amavis.bats

```diff
@@ -41,10 +45,9 @@ function teardown_file() {
   _run_in_container grep -F '127.0.0.1:10025' /etc/postfix/master.cf
   assert_success
 
-  _run_in_container_bash '[[ -f /etc/cron.d/amavisd-new.disabled ]]'
-  assert_failure
-  _run_in_container_bash '[[ -f /etc/cron.d/amavisd-new ]]'
+  _run_in_container_bash '[[ ! -f /etc/cron.d/amavisd-new.disabled ]]'
```

Comment:
Might warrant a `_file_should_not_exist_in_container`? 🤔 (_along with the other related checks in the file_)

Reply:
I was trying to keep the diff small so you wouldn't complain about this not belonging in this PR 😆😆😆 but I'll happily add `_file_should_not_exist_in_container` :D

Reply:
Done in [c593999](https://github.com/docker-mailserver/docker-mailserver/pull/3820/commits/c593999da8e5c24c7be8d7f1a406977c4f7fdb08)

Reply:
> so you wouldn't complain about this not belonging in this PR 😆😆😆

You were already halfway there with the inclusion anyway, and since I'm probably the only one providing a full review and we're at the end I don't mind as much 😅 (_but it should have all been handled separately_ **shakes fist** 😤 😝 )